cpp/src/io/orc/orc_gpu.hpp

```diff
@@ -286,10 +286,9 @@ void PostDecompressionReassemble(CompressedStreamInfo* strm_info,
 void ParseRowGroupIndex(RowGroup* row_groups,
                         CompressedStreamInfo* strm_info,
                         ColumnDesc* chunks,
-                        uint32_t num_columns,
-                        uint32_t num_stripes,
-                        uint32_t num_rowgroups,
-                        uint32_t rowidx_stride,
+                        size_type num_columns,
+                        size_type num_stripes,
+                        uint64_t rowidx_stride,
```

Comment:
Can you help me understand why the size of `rowidx_stride` has been increased?

Reply:
Because I made a mistake :D Fixed now, using size_type for this across the code.